cli/aws-cron-validator.ts

```diff
@@ -0,0 +1,109 @@
+class AWSCronError extends Error {}
+
+export class AWSCronValidator {
+
+    private static minuteExp = `(0?[0-9]|[1-5][0-9])`; // [0]0-59
```

Comment:
nit: consider moving these constants to an external file to improve readability in here.